packages/core/src/schema/models/evaluationTemplates.ts

```diff
@@ -0,0 +1,15 @@
+import { bigserial, text, varchar } from 'drizzle-orm/pg-core'
+
+import { latitudeSchema } from '..'
+import { timestamps } from '../schemaHelpers'
+
+export const evaluationTemplates = latitudeSchema.table(
+  'evaluations_templates',
+  {
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
+    name: varchar('name', { length: 256 }).notNull(),
+    description: text('description').notNull(),
+    prompt: text('prompt').notNull(),
+    ...timestamps(),
+  },
```

Comment:
**Evaluation Templates** — Library of our predefined evaluation templates. Users can either create a new evaluation from scratch, or just select one from our library.

Reply:
How do we track template changes over time? Should we have template versions? 

(I'm joking just wanted to annoy you)